docs/Test-Strategy-Info.mdx

```diff
@@ -1,31 +1,168 @@
+
+![Image](https://blog.getmason.io/content/images/2020/11/Testing-pyramid--6--1.jpg)
+Image [Source](https://blog.getmason.io/content/images/2020/11/Testing-pyramid--6--1.jpg)
+To avoid confusion, we will clarify what the test levels mean in our context.
+
+### Unit tests
+We use jest to write unit tests
+They normally exercise a single component (UI components, hooks, helper methods)
+Some examples of our unit tests can be found [here](https://github.com/bbc/simorgh/tree/bfbed54d2b1fe880e8f171023054c16d1c274ca2/src/app/components/Heading).
+Some patterns:
+
+- We use data-driven tests sometimes like [here](https://github.com/bbc/simorgh/blob/bfbed54d2b1fe880e8f171023054c16d1c274ca2/src/app/hooks/useImageColour/index.test.js#L10) to define inputs to tests
+- We use snapshot tests to easily test some cases like [here](https://github.com/bbc/simorgh/blob/4f385d94bee96af48eff2eca49c24cf382a3a494/src/app/components/FrostedGlassPromo/index.test.tsx#L76)
+    
+
+### Component tests
+We conduct visual regression testing using Chromatic, which alongside our unit tests, comprise our component tests layer.
+
+### Integration tests
+These test the integration between the different components to ensure a page is rendered properly. 
+They typically check that what’s rendered matches SIMORGH_DATA in a flexible generic way, i.e. check that a page renders “most read” topics but not the exact topics rendered, and that each topic has an image and header but not the content of either.
+Examples can be found in  [src/integration](https://github.com/bbc/simorgh/tree/bfbed54d2b1fe880e8f171023054c16d1c274ca2/src/integration).
+Check [here](https://github.com/bbc/simorgh/tree/latest/src/integration) for more details about our integration tests.
+
+### UI (E2E) tests
+These tests run the actual UI in a browser, and can check that everything renders correctly. For example, an integration test might check that “most read” topics are rendered with the correct HTML tags, but that does not necessarily mean they appear correctly (some CSS value might hide them). With the UI tests, we check that everything shows as expected in the UI.
+Examples and instructions can be found in [cypress](https://github.com/bbc/simorgh/tree/latest/cypress).
+
+
+
+> 🔊 discussion point with team: should we break UI / e2e tests as @Simon F comment mentions - I agree with the comment, but I am not sure this reflects what we do in our team right now?
+
+
+#### What levels to prioritise?
+
+Unit tests responsibility and scope are relatively easy to define: we already have a high level of code coverage and developers should aim to maintain that. 
+
+For integration and e2e tests, the general rule of thumb should be to only add an e2e test when an integration test is not enough to ensure coverage. Integration tests are less expensive and more stable and they should be prioritised.
```

Comment:
Should we give an example of when an integration test is not enough to ensure coverage - usually this is some form of interactivity, navigtion or running asynchronous scripts on the page. 

Perhaps we could also link to the section of the integration docs which mention this - there's a bit here https://github.com/bbc/simorgh/blob/latest/src/integration/README.mdx#i-already-wrote-unit-tests-for-a-component-do-i-need-to-write-an-integration-test-for-this-component but not sure if that's enough

```suggestion
For integration and E2E tests, the general rule of thumb should be to only add an E2E test when an integration test is not enough to ensure coverage. Integration tests are less expensive and more stable and they should be prioritised.
```